app/code/Magento/Sales/view/frontend/templates/reorder/sidebar.phtml

```diff
@@ -33,15 +34,17 @@
                                    data-bind="attr: {
                                         id: 'reorder-item-' + id,
                                         value: id,
-                                        title: is_saleable ? '<?= $block->escapeHtml(__('Add to Cart')) ?>' : '<?= $block->escapeHtml(__('Product is not salable.')) ?>'
+                                        title: is_saleable ?
+                                        '<?= $block->escapeHtml(__('Add to Cart')) ?>' :
+                                        '<?= $block->escapeHtml(__('Product is not salable.')) ?>'
                                    },
                                    disable: !is_saleable"
                                    class="checkbox" data-validate='{"validate-one-checkbox-required-by-name": true}'/>
                         </div>
                     </div>
                     <strong class="product-item-name">
                         <a data-bind="attr: {href: url}" class="product-item-link">
-                            <span data-bind="text: name"></span>
+                            <span data-bind="html: name"></span>
```

Comment:
Apparently this output should be escaped.
I'm not sure we should apply this change from the security perspective.

Reply:
Hi @dmytro-ch  

If We will not apply this change it shows error in frontend

Thanks